Blish HUD/GameServices/Gw2Mumble/PlayerCamera.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using Gw2Sharp.Mumble;
+using Microsoft.Xna.Framework;
+
+namespace Blish_HUD.Gw2Mumble {
+    public class PlayerCamera {
+
+        private readonly Gw2MumbleService _service;
+
+        private Vector3 _position    = Vector3.Zero;
+        private Vector3 _forward     = Vector3.Forward;
+        private float   _fieldOfView = 1;
+
+        private Matrix _view;
+        private Matrix _playerView;
+        private Matrix _projection;
+        private Matrix _worldViewProjection;
+
+        /// <summary>
+        /// The closest distance entities are rendered.
+        /// </summary>
+        public float NearPlaneRenderDistance { get; set; } = 0.01f;
+
+        /// <summary>
+        /// The farthest distance entities are rendered.
+        /// </summary>
+        public float FarPlaneRenderDistance  { get; set; } = 1000.0f;
```

Comment:
Do you want modules to be able to set this? Maybe we want an internal set and have it exposed somewhere if needed.

Reply:
I personally have no preference either way.  It was a holdover from the original `CameraService` and was kind of left like that in case any modules wanted to play with them.  

I can't think of any strong use cases for doing that, though, so for now I've removed the public setters on them and if we ever want to make them accessible down the road we can add them back or provide some other method of exposing them without causing a breaking change.